src/OneWayMirror.Core/OneWayMirror.cs

```diff
@@ -72,6 +79,59 @@ public override string ToString()
             _confirmBeforeCheckin = confirmBeforeCheckin;
             _lockWorkspacePath = lockWorkspacePath;
             _commitPortUtil = new CommitPortUtil(repository, workspace, workspacePath, host);
+
+            _userNamesMapFullPath = Path.Combine(tfsWorkspacePath, _userNamesMapRelativeFilePath);
+            _lazyUserNamesMap = null;
+            _lastUpdatedTime = default(DateTime);
+        }
+
+        private Dictionary<string, string> GetOrCreateGitToTfsUserNamesMap()
+        {
+            try
+            {
+                var lastUpdatedTime = File.GetLastWriteTime(_userNamesMapFullPath);
+                if (_lazyUserNamesMap != null && lastUpdatedTime == _lastUpdatedTime)
+                {
+                    return _lazyUserNamesMap;
+                }
+
+                var map = new Dictionary<string, string>(StringComparer.InvariantCultureIgnoreCase);
+                var lines = File.ReadAllLines(_userNamesMapFullPath);
+                foreach (var line in lines)
+                {
+                    if (line.StartsWith("#"))
+                    {
+                        // ignore comments
+                        continue;
+                    }
+
+                    var parts = line.Split(',');
+                    if (parts.Length >= 2)
+                    {
+                        var gitUserName = parts[0].TrimStart(' ');
+                        var tfsUserName = parts[1].TrimStart(' ');
+                        
+                        var tfsUserInfo = tfsUserName;
+                        if (parts.Length > 2)
+                        {
+                            var emailId = parts[2].TrimStart(' ');
+                            tfsUserInfo = string.Format("{0} ({1})", tfsUserName, emailId);
+                        }
+
+                        map.Add(gitUserName, tfsUserInfo);
+                    }
+                }
+
+                _lastUpdatedTime = lastUpdatedTime;
+                _lazyUserNamesMap = map;
+                
+                return map;
+            }
+            catch (Exception)
+            {
+                // Ignore exceptions.
```

Comment:
Lets log the exception.  I agree it's non-fatal but would be nice to see in the server output.